standard/standard_8.php

```diff
@@ -339,6 +341,16 @@ function ob_get_level(): int {}
  * <tr><td>blocksize</td><td>...</td></tr>
  * </table>
  */
+#[ArrayShape([
+    "level" => "int",
+    "type" => "int",
+    "status" => "int",
```

Comment:
```suggestion
    "flags" => "int",
```
https://github.com/php/php-src/blob/b9e895bca0a479cfebc63c782f556b016ea07f19/main/output.c#L1191